demo/index.html

```diff
@@ -2,24 +2,54 @@
 <html>
 <head>
   <!--[if lt IE 9]>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.2/html5shiv-printshiv.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/4.1.7/es5-shim.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/4.1.7/es5-sham.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.js"></script>
   <![endif]-->
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
   <title>Demo</title>
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <style type="text/css">
+    * {
+      box-sizing: border-box;
+    }
+    .tile {
+      display: block;
+      float: left;
+      width: 100px;
+      height: 100px;
+      margin: 20px;
+    }
+    .tile img {
+      display: block;
+      width: 100px;
+      height: 100px;
+    }
+    .demo {
+      width: 80%;
+      margin: auto;
+      position: relative;
+    }
+  </style>
 </head>
 <body>
   <!--[if lt IE 8]>
     <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
   <![endif]-->
-  <div id="content">
-    <h1>If you can see this, something is broken (or JS is not enabled)!!.</h1>
-  </div>
-  <script type="text-javascript" src="http://localhost:3000/webpack-dev-server.js"></script>
-  <script type="text/javascript" src="assets/main.js"></script>
+  <div id="content"></div>
+  <!--[if lt IE 9]>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/4.1.15/es5-shim.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/4.1.15/es5-sham.min.js"></script>
+  <![endif]-->
```

Comment:
This is correct as `es5-shim|sham` must come before _all other JS_ that needs any ES5 polyfills, but may be anywhere on the page otherwise.